rpm/gating.yaml

```diff
@@ -0,0 +1,12 @@
+--- !Policy
+product_versions:
+  - fedora-*
```

Comment:
The `gating.yaml` I never found a good documentation on it, let me know if you have any.